packages/image_picker/image_picker_platform_interface/lib/src/types/picked_file/html.dart

```diff
@@ -27,6 +27,8 @@ class PickedFile extends PickedFileBase {
 
   Future<Uint8List> get _bytes async {
     if (_initBytes != null) {
+      // See https://github.com/flutter/flutter/issues/143110
```

Comment:
This should be a TODO to do the migration, with a link to an issue that stays open and describes the migration needed and information about what release the replacement is available in. The goal isn't to leave `ignore`s indefinitely, but to clearly track the need to remove them and the timeline for doing so.

Reply:
Filed https://github.com/flutter/flutter/issues/143113 and updated the PR

PTAL